openapi/components/examples/friends/200-get-own-friend-requests.json

```diff
@@ -0,0 +1,16 @@
+{
+  "value": {
+    "incoming": [
+      "0xad0a8680845B19f833e317F00CCE41967c85253a",
```

Comment:
what about transforming each item into an object? Maybe you need more information, for example the timestamp.